tests/integration/compose/docker-compose-internal-auth_test.go

```diff
@@ -227,30 +236,42 @@ func (s *commonServicesInternalAuthDockerComposeUpTest) TestDockerComposeUp() {
 			// In golang, we cannot mix strings with string slice unpacking.
 			// Let's create a slice that will later be unpacked and used as an argument
 			// to the variadic function `docker.RunDockerCompose` parameter `args`.
+			argsConfig := []string{}
 			argsUp := []string{}
 			argsDown := []string{}
 			args := []string{"-f", testCase.composeFile}
 
-			for _, overrideFile := range s.overrideFiles {
+			for _, overrideFile := range testCase.overrideFiles {
 				args = append(args, "-f", overrideFile)
 			}
 
-			argsUp = append(args, "up", "--detach")
-			argsDown = append(args, "down", "--remove-orphans", "--timeout", "2")
+			argsConfig = append(args, "config")
+			argsUp = append(argsUp, args...)
+			argsUp = append(argsUp, "up", "--detach")
+			argsDown = append(argsDown, args...)
+			argsDown = append(argsDown, "down", "--remove-orphans", "--timeout", "2")
```

Comment:
We need to *copy* the slices. Then we can modify them. Prior to this change, we were modifying the `args` slice.  Oops.